cmake/version.cmake

```diff
@@ -56,3 +84,18 @@ unset(MAJOR)
 unset(MINOR)
 unset(PATCH)
 unset(PROJECT_VERSION_WITHOUT_TWEAK)
+
+if(CMAKE_VERBOSE_MAKEFILE) # OR 1)
+message(STATUS "
+  PROJECT_VERSION=${PROJECT_VERSION}
+
+  KERNEL_VERSION_STRING: ${KERNEL_VERSION_STRING}
+  KERNEL_VERSION_MAJOR: ${KERNEL_VERSION_MAJOR}
+  KERNEL_VERSION_MINOR: ${KERNEL_VERSION_MINOR}
+  KERNEL_PATCHLEVEL: ${KERNEL_PATCHLEVEL}
+
+  KERNEL_VERSION_NUMBER: ${KERNEL_VERSION_NUMBER}
+  ZEPHYR_VERSION_CODE: ${ZEPHYR_VERSION_CODE}
+  KERNELVERSION: ${KERNELVERSION}
+")
```

Comment:
I don't know about this.

Is the intention to be able to debug? If so, isn't printf-debugging more appropriate?

Reply:
By "printf-debugging" do you mean: only locally, nothing in git? So this is of course what I did at first and  I found it much more tedious than usual because of:
- the number of variables and amount of typing involved
- the number of local variables and the time required to weed these out

Reply:
This should be dropped, the noise-to-value ratio does not justify it.

Users that want to see the values of variables can either run cmake under --trace-expand or add printf's.

Reply:
OK will drop this. This code was useful to "bootstrap" the sample output at the top but less so now.